app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+pages:
+  page_load:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long each page takes to load.
```

Comment:
Please be a bit more specific about what you are measuring: does this include DNS lookup? What does "page load" mean? How is a page considered fully loaded? If you are unsure, you can mention that this relies on the `onPageStart` and `onPageStop` from GeckoView. You can also link to them using Markdown: this will be linkified when generating the docs.

Reply:
I am going to move this to anther PR because of blocker issues.